test/integration/list/test_list.py

```diff
@@ -0,0 +1,107 @@
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+
+import yaml
+
+
+def test_list_enabled_apps(tt_cmd):
+    test_app_path_src = os.path.join(os.path.dirname(__file__), "multi_app")
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        test_app_path = os.path.join(tmpdir, "multi_app")
+        shutil.copytree(test_app_path_src, test_app_path)
+
+        # List all instances.
+        start_cmd = [tt_cmd, "list"]
+        instance_process = subprocess.Popen(
+            start_cmd,
+            cwd=test_app_path,
+            stderr=subprocess.STDOUT,
+            stdout=subprocess.PIPE,
+            text=True
+        )
+        start_output = instance_process.stdout.read()
+        assert re.search("app1", start_output)
```

Comment:
Missing instances list check.

Reply:
I didn't understand your comment. Can you pls clarify it.

Reply:
As I can see `app1` is a multi-instance app. If yes, need to check that instances names are in output.

Reply:
Okey, sure. Added.